AppConfig.js

```diff
@@ -91,6 +91,11 @@ var config = {
         name: 'Yahoo',
```

Comment:
We don't need `yahoofinance` anymore? Should we remove it?

Reply:
Fixed